### PR TITLE
AG-10131 Apply darkmode to custom theme examples

### PR DESCRIPTION
**Pull request by jacobp100**
https://ag-grid.atlassian.net/browse/AG-10131

### CODE REVIEW

**Review by alantreadway — APPROVED** [review]
LGTM

File: packages/ag-charts-website/src/design-system/modules/GalleryExampleLink.module.scss [hunk 1]
```diff
@@ -8,37 +8,29 @@
...
-        border: $border-width solid var(--color-border-primary);
-        opacity: 0;
-        transition: opacity 0.5s ease-in-out;
-    }
+    transition: border-color 0.5s ease-in-out;
 
-    &:hover::after {
+    &:hover {
         opacity: 1;
+        --link-border-color: var(--color-border-primary);
```
Comment (markdurrant) [inline packages/ag-charts-website/src/design-system/modules/GalleryExampleLink.module.scss:16]: Minor stylistic thing. I usually try and put Sass/CSS variables at the top of the ruleset with a space underneath. Something like 

```
&:hover {
    --link-border-color: var(--color-border-primary);
    
    opacity: 1;
}
```

**Review by markdurrant — APPROVED** [review]
All LGTM 👍 One very minor stylistic comment.

**Review by taktran — APPROVED** [review]
👍🏼

**Pull request merged**